tasks/scripts/concourse-build

```diff
@@ -6,15 +6,13 @@ set -e -x
 export GOPATH=$PWD/gopath
 export PATH=$PWD/gopath/bin:$PATH
 
-if [ "$#" -lt 1 ]; then
-  echo "usage: $0 <platform>" >&2
+if [ -z "${PLATFORM}" ]; then
+  echo "usage: export PLATFORM=<platform> $0" >&2
```

Comment:
```suggestion
  echo "usage: PLATFORM=<platform> $0" >&2
```

the shell will already take care of placing the environment variable for the executable being run